src/ralph/backends/data/es.py

```diff
@@ -111,6 +111,18 @@ class ESQuery(BaseQuery):
     track_total_hits: Literal[False] = False
 
 
+class ESQuery(BaseESQuery):
+    """Elasticsearch query model.
+
+    Attributes:
+        query_string (str): Same as in `BaseESQuery`, however also accepts a JSON
+            instance of `BaseESQuery` which, if provided, is used instead.
+    """
+
+    # pylint: disable=unsubscriptable-object
+    query_string: Union[Json[BaseESQuery], str, None]
```

Comment:
`Json[BaseESQuery], str` causes to silently fall back using a Lucene query if the query is not in JSON format.
We probably want to add a log message when this happens.